crates/wasi/src/p3/filesystem/mod.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{DirPerms, FilePerms, OpenMode};
+
+// TODO: implement
+pub struct Dir;
+
+impl Dir {
+    #[allow(unused)]
```

Comment:
Mind changing this to `#[expect(...)]`? (that way it forces removal of the attribute once it's used)